src/workspace-element.js

```diff
@@ -56,10 +56,16 @@ class WorkspaceElement extends HTMLElement {
   }
 
   updateGlobalTextEditorStyleSheet () {
-    const styleSheetSource = `atom-text-editor {
-  font-size: ${this.config.get('editor.fontSize')}px;
-  font-family: ${this.config.get('editor.fontFamily')};
-  line-height: ${this.config.get('editor.lineHeight')};
+    const styleSheetSource = `atom-workspace {
+  --editor-font-size: ${this.config.get('editor.fontSize')}px;
+  --editor-font-family: ${this.config.get('editor.fontFamily')};
+  --editor-line-height: ${this.config.get('editor.lineHeight')};
+}
+
+atom-text-editor {
+  font-family: var(--editor-font-family);
```

Comment:
We could add a fallback, so that while you type in the settings or mispell, it doesn't show `Times New Roman` and make the cursor jump. 

![font](https://user-images.githubusercontent.com/378023/36297541-a1095554-1336-11e8-968e-291a5f15c51d.gif)

Maybe just `font-family: var(--editor-font-family), monospace;`

![font2](https://user-images.githubusercontent.com/378023/36297593-f18edee0-1336-11e8-8bfc-b838afc84e1b.gif)

Reply:
![2018-02-16 00_16_30](https://user-images.githubusercontent.com/755844/36298792-b472417e-12ae-11e8-8e4e-6c941e325140.gif)

:)